docs/typescript.md

```diff
@@ -17,29 +17,29 @@ To get started with Mongoose in TypeScript, you need to:
 import { Schema, model, connect } from 'mongoose';
 
 // 1. Create an interface representing a document in MongoDB.
-interface IUser {
+interface User {
   name: string;
   email: string;
   avatar?: string;
 }
 
 // 2. Create a Schema corresponding to the document interface.
-const userSchema = new Schema<IUser>({
+const userSchema = new Schema<User>({
   name: { type: String, required: true },
   email: { type: String, required: true },
   avatar: String
 });
 
 // 3. Create a Model.
-const User = model<IUser>('User', userSchema);
+const UserModel = model<User>('User', userSchema);
 
 run().catch(err => console.log(err));
 
 async function run() {
   // 4. Connect to MongoDB
   await connect('mongodb://localhost:27017/test');
 
-  const user = new User({
+  const user = UserModel.create({
```

Comment:
Well, this is not equivalent to the previous code. Why did you change it?

Reply:
Please undo this change, should be `const user = new UserModel({ ...`. `UserModel.create()` is async.

Reply:
```suggestion
  const user = new UserModel({
```